.gitignore

```diff
@@ -186,11 +188,8 @@ _build
 /runtime/ocamlrund
 /runtime/ocamlruni
 /runtime/ld.conf
-/runtime/interp.a.lst
-/runtime/*.[sd]obj
 /runtime/.gdb_history
-/runtime/*.d.c
-/runtime/*.pic.c
```

Comment:
I think these are legacy from before the merge of the runtime directories, @shindere?